packages/number-field/test/number-field.test.ts

```diff
@@ -566,7 +566,7 @@ describe('NumberField', () => {
                 ],
             });
             expect(inputSpy.callCount).to.equal(4);
-            expect(changeSpy.callCount).to.equal(1);
+            expect(changeSpy.callCount).to.equal(0); // the actual value hasn't changed so changespy won't be called
```

Comment:
Great point! The name of this test is no longer valid.

We may want both this test and the previous test, just make sure it doesn't end on the same value as it started...

Reply:
Also...

```suggestion
            expect(changeSpy.callCount, 'value does not change from initial value so no "change" event is dispatched').to.equal(0);
```

The `expect` method accepts "comments" as the second argument.